src/ledger-bridge.ts

```diff
@@ -26,14 +26,29 @@ export type LedgerSignTypedDataResponse = Awaited<
   ReturnType<LedgerHwAppEth['signEIP712HashedMessage']>
 >;
 
+export type LedgerBridgeOptions = Record<string, string | number | object>;
+
+export type LedgerBridgeSerializeData = Record<
+  string,
+  string | number | object
+>;
```

Comment:
This type is quite broad, and the specific Bridge doesn't have any way to narrow this type. 

It is a bit related to my previous question regarding the relationship / difference between `LedgerBridgeOptions` and `LedgerBridgeSerializeData`, but ideally these two should be narrowed to a smaller set of possible valid values depending on the bridge that we are interacting with